app/controllers/ViewTransfersController.scala

```diff
@@ -16,40 +18,22 @@ import javax.inject.Inject
 class ViewTransfersController @Inject() (val consignmentService: ConsignmentService, val keycloakConfiguration: KeycloakConfiguration, val controllerComponents: SecurityComponents)
     extends TokenSecurity {
 
-  private val inProgressStep = "InProgress"
-  private val failedStep = "Failed"
-  private val completedWithIssuesStep = "CompletedWithIssues"
-  private val completedStep = "Completed"
-
-  private val transferStatusInProgress = "In Progress"
-  private val transferStatusFailed = "Failed"
-  private val transferStatusTransferred = "Transferred"
-
-  private val resumeTransfer = "Resume transfer"
-  private val viewErrors = "View errors"
-  private val contactUs = "Contact us"
-  private val downloadReport = "Download report"
-  private val view = "View"
-
-  private def standardTransferPageNameToUrl(consignmentId: UUID) = Map(
-    "series" -> routes.SeriesDetailsController.seriesDetails(consignmentId).url,
-    "taPrivateBeta" -> routes.TransferAgreementPrivateBetaController.transferAgreement(consignmentId).url,
-    "taCompliance" -> routes.TransferAgreementComplianceController.transferAgreement(consignmentId).url,
-    "upload" -> routes.UploadController.uploadPage(consignmentId).url,
-    "fileChecks" -> routes.FileChecksController.fileChecksPage(consignmentId).url,
-    "fileChecksResults" -> routes.FileChecksResultsController.fileCheckResultsPage(consignmentId).url,
-    "export" -> routes.DownloadMetadataController.downloadMetadataCsv(consignmentId).url
-  )
-
-  private def judgmentTransferPageNameToUrl(consignmentId: UUID) = Map(
-    "beforeYouUpload" -> routes.BeforeUploadingController.beforeUploading(consignmentId).url,
-    "upload" -> routes.UploadController.judgmentUploadPage(consignmentId).url,
-    "fileChecks" -> routes.FileChecksController.judgmentFileChecksPage(consignmentId).url,
-    "fileChecksResults" -> routes.FileChecksResultsController.judgmentFileCheckResultsPage(consignmentId).url,
-    "export" -> routes.TransferCompleteController.judgmentTransferComplete(consignmentId).url
-  )
-
-  val statusColours: Map[String, String] = Map(transferStatusInProgress -> "yellow", transferStatusFailed -> "red", contactUs -> "red", transferStatusTransferred -> "green")
+  private val statusColours: Map[String, String] = Map(InProgress.value -> "yellow", Failed.value -> "red", ContactUs.value -> "red", Transferred.value -> "green")
+  private val formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm:ss")
+
+  implicit class ConsignmentStatusesHelper(statuses: List[ConsignmentStatuses]) {
+    def containsStatus(statusType: StatusType): Boolean = {
```

Comment:
This might make things more complicated but the case statement on line 77 is getting quite crowded. I know it's a complicated thing so it needs to be but I wonder if making this take multiple statuses would help. 
So change the method to something like
```scala
    def containsStatuses(statusTypes: StatusType*): Boolean = {
      statusTypes.foldLeft(false)((contains, statusType) => contains || statuses.exists(_.statusType == statusType.id))
    }
```
Then in the case statement, instead of having 
```scala
s.containsStatus(ServerAntivirusType) || s.containsStatus(ServerChecksumType) || s.containsStatus(ServerFFIDType)
``` 
you could do 
```scala
containsStatuses(ServerAntivirusType, ServerChecksumType, ServerFFIDType)
```
I'm not convinced this would help the readability at all and it might make it worse. What do you reckon?

Reply:
That does help